modules/installation-osp-deploying-all-bare-metal-preexisting.adoc

```diff
@@ -0,0 +1,70 @@
+[id="installation-osp-deploying-all-bare-metal-preexisting_{context}"]
+= Configuring a cluster to use bare-metal control plane and compute machines
+
+If you want your cluster to use bare-metal control plane and compute machines, modify the
+`install-config.yaml` file. In this configuration, control plane and compute machines are
+deployed on user-provisioned infrastructure.
+
+[NOTE]
+====
+Bare-metal compute machines are not supported on clusters that use Kuryr.
+====
+
+.Prerequisites
+
```

Comment:
Do we want to be explicit about the current lack of support for provider network?

Reply:
Works for me. Just here, or in other modules, too? @mandre

Reply:
This is also applicable to other modules.

Reply:
TY. Addressed in https://github.com/openshift/openshift-docs/pull/25167/commits/4cb16edca724497e56a173a517304dbfcab9ef40, I think. @mandre